src/microengineclamav/scan.py

```diff
@@ -17,10 +17,17 @@ def scan(bounty: Bounty) -> ScanResult:
     content = bounty.fetch_artifact()
     # No need to close this. Each connection is opened and closed in each method
     clamd_socket = clamd.ClamdNetworkSocket(settings.CLAMD_HOST, settings.CLAMD_PORT, settings.CLAMD_TIMEOUT)
-    result = clamd_socket.instream(io.BytesIO(content))
-    stream_result = result.get('stream', [])
+    try:
+        vendor = clamd_socket.version()
+        result = clamd_socket.instream(io.BytesIO(content))
+    except clamd.ConnectionError:
+        logger.exception('Error connecting to clamd')
+        raise errors.CalledProcessScanError('Unable to connect')
+    except clamd.ResponseError:
+        logger.exception('Error in clamd')
+        raise errors.CalledProcessScanError('Bad response')
```

Comment:
```suggestion
    try:
        vendor = clamd_socket.version()
        result = clamd_socket.instream(io.BytesIO(content))
    except clamd.ConnectionError as err:
        logger.exception('Error connecting to clamd')
        raise errors.CalledProcessScanError('Unable to connect') from err
    except clamd.ResponseError as err:
        logger.exception('Error in clamd')
        raise errors.CalledProcessScanError('Bad response') from err
```

I suggest to `raise from` to get a traceback explicit about _not having an error inside another_.

See:
- using only `raise`:
```python
>>> try:
...     1/0
... except ZeroDivisionError as err:
...     raise RuntimeError()
... 
Traceback (most recent call last):
  File "<ipython-input-4-a93299a09d29>", line 2, in <module>
    1/0
ZeroDivisionError: division by zero

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "<ipython-input-4-a93299a09d29>", line 4, in <module>
    raise RuntimeError()
RuntimeError
```
- using `raise from`:
```python
>>> try:
...     1/0
... except ZeroDivisionError as err:
...     raise RuntimeError() from err
...
Traceback (most recent call last):
  File "<ipython-input-5-50d43a139639>", line 2, in <module>
    1/0
ZeroDivisionError: division by zero

The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "<ipython-input-5-50d43a139639>", line 4, in <module>
    raise RuntimeError() from err
RuntimeError
```